### PR TITLE
D3

### PR DESCRIPTION
**Pull request by cuestalvaro**

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -6,5 +6,21 @@ def show
...
\ No newline at end of file
+    def data
+      array = [] 
+      current_user.tracks.all.each do |track|
+         array << {
+          task: track.name,
+          hour: calchs(track)
+        } 
+      end
+      render json: array.to_a.reject(&:empty?).to_json
```
Comment (mrplum) [inline app/controllers/users_controller.rb]: applying rubbish 

```
render json: current_user.tracks.to_a.map do |track|
    {
           task: track.name,
           hour: calchs(track)
         }
       end.reject(&:empty?).to_json
```

Could you try that? It's the same but more in the ruby way

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -6,5 +6,21 @@ def show
...
+    end
+
+    private
+      def calchs (track)
+        hs = ((track.ends_at - track.starts_at) / 3600).floor
```
Comment (mrplum) [inline app/controllers/users_controller.rb]: instead to use `3600` use `1.hour`

instead to use `60` use `1.minute` 

but I think that you try to calculate the difference between two date times and return the float of that... welcome to ruby lang, it's just `((track.ends_at - track.starts_at) * 1.day).to_i`

I think that will be better express it in integers like the above method. 

Also, is very easy to do a test of this method, could you try? It needs a track with an `ends_at` and a `starts_at` that you know, then use the method and expect the result that you calculate manually 

for example...
```
[80] pry(main)> start
=> Tue, 12 Jan 2010 15:00:00 +0000
[81] pry(main)> ends
=> Tue, 12 Jan 2010 17:20:00 +0000
[82] pry(main)> (re*1.day).to_i
=> 8400
```

if you know that start and ends at differs in 2:20hs, then you could expect `8400` seconds

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (cuestalvaro) [reply app/controllers/users_controller.rb]: listo

File: app/controllers/users_controller.rb (hunk 2 above)
Comment (cuestalvaro) [reply app/controllers/users_controller.rb]: listo

**Pull request merged**